index.js

```diff
@@ -14,6 +14,10 @@
                          loadScript("tests/index.js"));
   }
 
+  if (navigator.userAgent.indexOf('Chrome')) {
+    loadingPromises.push(loadScript("chrome_polyfills.js"));
+  }
```

Comment:
@brendandahl This loads chrome_polyfills.js in Firefox too, because navigator.userAgent.indexOf('Chrome') == -1, which is truthy, if "Chrome" isn't in the user-agent string.

I'm still not sure why that causes this to fail, but tests/contacts.js also sets navigator.mozContacts to a new object, so perhaps the Chrome polyfill overrides that in a way that busts some test.

So I think there are two things to do here:

1. correct the conditional, so we only load the polyfill in Chrome;
2. make the polyfill also conditional on `midletClassName !== "RunTests"`, so we don't override the test polyfill when running tests in Chrome (if that even works).